src/main/java/pl/task/parkingmeter/controller/VehicleController.java

```diff
@@ -0,0 +1,190 @@
+package pl.task.parkingmeter.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
```

Comment:
importy z gwizadkę utrudniają review

Reply:
to automatyczny refactor IntelliJ, przy większej ilości importów zastepuje je w ten sposób, zmienię w ustawieniach i wprowadzę później zmiany